### PR TITLE
zh-translation: /blog/2019/performance-best-practices/index.md #1134

### PR DESCRIPTION
**Pull request by gorda**
Ref: [ServiceMesher#1134](https://github.com/servicemesher/istio-official-translation/issues/1134)

### CODE REVIEW

File: content/zh/blog/2019/performance-best-practices/index.md [hunk 1]
```diff
@@ -1,63 +1,63 @@
...
 
-Lastly, our test environment measures requests between two pods, not many. The client pod is [Fortio](http://fortio.org/), which sends traffic to the server pod.
+最后，我们的测试环境可以测量两个 pod 之间少量的请求。客户端 pod 是 [Fortio](http://fortio.org/)，它将流量发送到服务端 pod。
 
-Why test with only two pods? Because scaling up throughput (RPS) and connections (threads) has a greater effect on Envoy's performance than increasing the total size of the service registry — or, the total number of pods and services in the Kubernetes cluster. When the size of the service registry grows, Envoy does have to keep track of more endpoints, and lookup time per request does increase, but by a tiny constant. If you have many services, and this constant becomes a latency concern, Istio provides a [Sidecar resource](/zh/docs/reference/config/networking/sidecar/), which allows you to limit which services each Envoy knows about.
+为什么只用两个吊舱测试？因为增加吞吐量（RPS）和连接（线程）对 Envoy 的性能的影响比增加服务注册表的总大小（或 Kubernetes 集群中 Pod 和服务的总数）更大。当服务注册表的大小增加时，Envoy 必须跟踪更多的端点，并且每个请求的查找时间确实增加了，但是增加了一个很小的常数。如果您有许多服务，并且此常数成为延迟问题，则 Istio 提供 [Sidecar 资源](/zh/docs/reference/config/networking/sidecar/)，它使您可以限制每个 Envoy 知道的服务。
```
Comment (ilylia) [inline content/zh/blog/2019/performance-best-practices/index.md]: 吊舱，pod

File: content/zh/blog/2019/performance-best-practices/index.md [hunk 2]
```diff
@@ -1,63 +1,63 @@
...
+总体而言，我们发现 Istio [sidecar 代理](/zh/docs/ops/deployment/architecture/#envoy) 的延迟取决于并发连接数。以每秒 1000 个请求（RPS）的速度，通过 16 个连接，Istio 延迟在 50% 时增加 **3 毫秒**，在 99% 时增加 **10 毫秒**。
 
-In the [Istio Tools repository](https://github.com/istio/tools/tree/3ac7ab40db8a0d595b71f47b8ba246763ecd6213/perf/benchmark), you’ll find scripts and instructions for measuring Istio's data plane performance, with additional instructions on how to run the scripts with [Linkerd](https://linkerd.io), another service mesh implementation. [Follow along](https://github.com/istio/tools/tree/3ac7ab40db8a0d595b71f47b8ba246763ecd6213/perf/benchmark#setup) as we detail some best practices for each step of the performance test framework.
+在[Istio Tools 库](https://github.com/istio/tools/tree/3ac7ab40db8a0d595b71f47b8ba246763ecd6213/perf/benchmark) 中，您将找到用于测量 Istio 数据平面性能的脚本和说明，以及有关如何使用另一服务网格实现 [Linkerd](https://linkerd.io) 运行脚本的其他说明。在我们详细介绍性能测试框架的每个步骤的一些最佳实践时，请[遵循](https://github.com/istio/tools/tree/3ac7ab40db8a0d595b71f47b8ba246763ecd6213/perf/benchmark#setup)。
```
Comment (ilylia) [inline content/zh/blog/2019/performance-best-practices/index.md]: 库，仓库

File: content/zh/blog/2019/performance-best-practices/index.md [hunk 3]
```diff
@@ -75,7 +75,7 @@ In the 90th percentile, with 16 concurrent connections, Istio adds **6ms**; with
     caption=""
```
Comment (ilylia) [inline content/zh/blog/2019/performance-best-practices/index.md]: alt 和 title 也需要翻译吧

Comment (gorda) [reply content/zh/blog/2019/performance-best-practices/index.md]: > 建议修改

@ilylia 感谢细心 review，相关建议已修改

**Pull request merged**